### PR TITLE
Update all plugins to use plugin/pkg/log

### PR DESCRIPTION
**Pull request by miekg**
I wish this could have been done with sed. Alas manually changed all
callers to use the new plugin/pkg/log package.

### CODE REVIEW

**Comment by miekg** [comment]
also dropping a bunch of log.SetOutput(ioutil.Discard) in tests.

File: plugin/dnstap/dnstapio/io.go [hunk 1]
```diff
@@ -102,16 +103,16 @@ func (dio *dnstapIO) flushBuffer() {
...
 	if err := dio.enc.flushBuffer(); err != nil {
-		log.Printf("[WARN] Connection lost: %s", err)
+		log.Warningf("Connection lost: %s", err)
 		dio.closeConnection()
 		if err := dio.newConnect(); err != nil {
-			log.Printf("[ERROR] Cannot connect to dnstap: %s", err)
+			log.Errorf("Cannot connect to dnstap: %s", err)
 		} else {
-			log.Print("[INFO] Reconnected to dnstap")
+			log.Error("Reconnected to dnstap")
```
Comment (johnbelamaric) [inline plugin/dnstap/dnstapio/io.go]: Should be Info.

**Review by johnbelamaric — CHANGES_REQUESTED** [review]
One minor tweak needed.

**Comment by miekg** [comment]
[ Quoting <notifications@github.com> in "Re: [coredns/coredns] Update all pl..." ]
>johnbelamaric requested changes on this pull request.
>
>One minor tweak needed.
>
>>  		} else {
>-			log.Print("[INFO] Reconnected to dnstap")
>+			log.Error("Reconnected to dnstap")
>
>Should be Info.

done

**Pull request merged**